### PR TITLE
New Package: Resolume.Arena version 7.15.0.23381

### PR DESCRIPTION
**Pull request by sandercox**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/110548&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/r/Resolume/Arena/7.15.0.23381/Resolume.Arena.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Created using wingetcreate 1.2.0.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Resolume.Arena
+PackageVersion: 7.15.0.23381
+PackageLocale: en-US
+Publisher: Resolume B.V.
```
Comment (Trenly) [inline manifests/r/Resolume/Arena/7.15.0.23381/Resolume.Arena.locale.en-US.yaml:7]:
```suggestion
Publisher: Resolume
```
This should match Control Panel

File: manifests/r/Resolume/Arena/7.15.0.23381/Resolume.Arena.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+PublisherUrl: https://resolume.com
+PublisherSupportUrl: https://resolume.com/support/
+PrivacyUrl: https://resolume.com/company/privacy
+Author: Resolume B.V.
+PackageName: Resolume Arena
+PackageUrl: https://resolume.com/download
+License: Proprietary
+Copyright: Copyright ©2001-2023 Resolume B.V. All rights reserved
+ShortDescription: Resolume Arena
```
Comment (Trenly) [inline manifests/r/Resolume/Arena/7.15.0.23381/Resolume.Arena.locale.en-US.yaml:16]:
```suggestion
ShortDescription: Similar to Resolume Avenue, but with advanced options for projection mapping and blending
```

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/r/Resolume/Arena/7.15.0.23381/Resolume.Arena.locale.en-US.yaml (hunk 1 above)
Comment (sandercox) [reply manifests/r/Resolume/Arena/7.15.0.23381/Resolume.Arena.locale.en-US.yaml]: Thanks for pointing this out.

File: manifests/r/Resolume/Arena/7.15.0.23381/Resolume.Arena.locale.en-US.yaml (hunk 2 above)
Comment (sandercox) [reply manifests/r/Resolume/Arena/7.15.0.23381/Resolume.Arena.locale.en-US.yaml]: Updated the description and short description

**Comment by sandercox** [comment]
Not sure what evidence is required, this is our cloudfront hosting URL for CDN availability. This is what the https://resolume.com/download options redirect you to.

File: manifests/r/Resolume/Arena/7.15.0.23381/Resolume.Arena.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.0.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Resolume.Arena
+PackageVersion: 7.15.0.23381
+Installers:
+- InstallerUrl: https://dd5sgwxv3xok.cloudfront.net/Resolume_Arena_7_15_0_rev_23381_Installer.exe
```
Comment (stephengillie) [inline manifests/r/Resolume/Arena/7.15.0.23381/Resolume.Arena.installer.yaml:7]:
```suggestion
- InstallerUrl: https://resolume.com/download/files?file=Resolume_Arena_7_15_0_rev_23381_Installer.exe
```

**Comment by stephengillie** [comment]
> Not sure what evidence is required, this is our cloudfront hosting URL for CDN availability. This is what the https://resolume.com/download options redirect you to.

Our robot couldn't tell that this was your official mirror, and needed human assistance. Thanks for your patience.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**